cdc/tests/integration_tests/_utils/start_tls_tidb_cluster_impl

```diff
@@ -45,26 +45,49 @@ cert-path = "$TLS_DIR/server.pem"
 key-path = "$TLS_DIR/server-key.pem"
 EOF
 
-echo "Starting TLS PD..."
+echo "Starting Upstream TLS PD..."
```

Comment:
There are some duplicated codes between creating upstream & downstram cluster, and between creationg non-tls & tls clusters.

It's OK now, but suggest to consider extracting some core codes later. E.g, a function to start an single cluster, with arguments such as host/port of PD/TiDB/TiKV, `enable_tls`, and `log_prefix`.